CMakeLists.txt

```diff
@@ -285,5 +291,7 @@ message("             boost system Library: ${DATAFED_BOOST_SYSTEM_LIBRARY_PATH}
 message("         boost filesystem Library: ${DATAFED_BOOST_FILESYSTEM_LIBRARY_PATH}")
 message("    boost program_options Library: ${DATAFED_BOOST_PROGRAM_OPTIONS_LIBRARY_PATH}")
 message("                     zlib Library: ${DATAFED_ZLIB_LIBRARIES}")
+message("                   gssapi Library: ${DATAFED_GSSAPI_LIBRARIES}")
+message("            globus_common Library: ${DATAFED_GLOBUS_COMMON_LIBRARIES}")
```

Comment:
Are these used at all outside of being printed?

Reply:
Yes, they are used when building authz tests. in the 1159-refactor-authz-cpp-file branch.

Reply:
This is just broken up into pieces to help merge piece meal.